packages/@holaplexui-react/src/components/Form.tsx

```diff
@@ -207,26 +201,26 @@ function FormSelectButton({
   children,
   icon,
   placeholder = 'select an option',
-  dropdown = <Icon.ChevronDown />
+  dropdown = <Icon.ChevronDown />,
 }: FormSelectButtonProps): JSX.Element {
   return (
     <Listbox.Button
-      className={clsx(
-        'w-full inline-block',
-        'form-select-button',
-        {
-          'pl-12': icon,
-        },
-      )}
+      className={clsx('w-full inline-block', 'form-select-button', {
+        'pl-12': icon,
+      })}
     >
-      {children ? <span>{children}</span> : <span className="form-select-button-placeholder">{placeholder}</span>}
+      {children ? (
```

Comment:
ok it already handles the children being undefined just adjust the prop types then to allow for undefined.